sci-physics/madgraph5/madgraph5-3.5.3.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# does not escape strings properly, so no python3_12 for now
+PYTHON_COMPAT=( python3_11 )
+inherit fortran-2 python-single-r1
+
+MY_PNN="MadGraph5"
+MY_PV=$(ver_rs 1-3 '_')
+MY_PN="MG5_aMC_v"
+MY_PF=${MY_PN}${MY_PV}
+
+DESCRIPTION="MadGraph5_aMC@NLO"
+HOMEPAGE="https://launchpad.net/mg5amcnlo"
+SRC_URI="https://launchpad.net/mg5amcnlo/$(ver_cut 1).0/$(ver_cut 1-2).x/+download/${MY_PN}${PV}.tar.gz -> ${MY_PNN}-${PV}.tar.gz"
+S="${WORKDIR}/${MY_PF}"
+
+LICENSE="UoI-NCSA"
+SLOT="3"
+KEYWORDS="~amd64"
+# TODO add pineapple, herwig, syscalc, pjfrym, pineappl
+IUSE=" +hepmc +lhapdf +fastjet pythia collier thepeg" # td madanalysis5 ninja samurai golem95
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	sys-libs/zlib
+	sys-devel/gcc:*[fortran]
+	$(python_gen_cond_dep '
+		dev-python/numpy[${PYTHON_USEDEP}]
+	')
+	lhapdf? ( sci-physics/lhapdf[${PYTHON_SINGLE_USEDEP}] )
+	fastjet? ( sci-physics/fastjet[${PYTHON_SINGLE_USEDEP}] )
+	pythia? ( sci-physics/pythia:8=[examples] )
+	hepmc? (
+		sci-physics/hepmc:2
+		sci-physics/hepmc:3
+	)
```

Comment:
We need both?

Reply:
It's own install just install hepmc2, but I hope they update it soon. I'll drop the :3 for now, but some dependencies might pull it in either way.